### PR TITLE
Rework configuration option explanation into a table format

### PR DESCRIPTION
**Pull request by settermjd**
For proper rendering, this change depends on #11 being merged.

### CODE REVIEW

**Comment by tboerger** [comment]
That got to be discussed, I personally prefer the list style. /cc @felixboehm @butonic @refs

File: docs/content/getting-started.md [hunk 1]
```diff
@@ -27,79 +27,62 @@ If you prefer to configure the service with environment variables you can see th
...
+| `HELLO_DEBUG_ADDR` | Address to bind debug server, defaults to `0.0.0.0:8390` |
+| `HELLO_DEBUG_TOKEN` | Token to grant metrics access, empty default value |
+| `HELLO_DEBUG_PPROF` | Enable pprof debugging, defaults to `false` |
+| `HELLO_HTTP_ADDR` | Address to bind http server, defaults to `0.0.0.0:8380` |
+| `HELLO_HTTP_ROOT` | Root path for http endpoint, defaults to `/` |
+| `HELLO_ASSET_PATH` | Path to custom assets, empty default value |
 
 ##### Health
 
-HELLO_DEBUG_ADDR
```
Comment (tboerger) [inline docs/content/getting-started.md]: Why have this been removed?

File: docs/content/getting-started.md [hunk 2]
```diff
@@ -27,79 +27,62 @@ If you prefer to configure the service with environment variables you can see th
...
+| `--http-addr` | Address to bind http server, defaults to `0.0.0.0:8380` |
+| `--http-root` | Root path for http endpoint, defaults to `/` |
+| `--asset-path` | Path to custom assets, empty default value |
 
---debug-token
-: Token to grant metrics access, empty default value
 
---debug-pprof
-: Enable pprof debugging, defaults to `false`
+| Setting     | Description |
```
Comment (tboerger) [inline docs/content/getting-started.md]: Have been added at the wrong headline? This is the flag for `Health`.

File: docs/content/getting-started.md [hunk 3]
```diff
@@ -129,101 +112,41 @@ ocis-hello health --help
...
-
-go_threads
-: Number of OS threads created
-
-promhttp_metric_handler_requests_in_flight
-: Current number of scrapes being served
 
-promhttp_metric_handler_requests_total
-: Total number of scrapes by HTTP status code
+| Setting      | Description |
```
Comment (tboerger) [inline docs/content/getting-started.md]: That's not a setting, it's the metric.

**Comment by refs** [comment]
> That got to be discussed, I personally prefer the list style. /cc @felixboehm @butonic @refs

I'm in for the table format. Yes the editing is ugly, but it's easier for the eye to follow it 🤷‍♂

**Comment by DeepDiver1975** [comment]
heavily conflicting -> needs to be restarted from scratch -> closing

**Pull request closed without merging**